specification/storage/data-plane/Microsoft.FileStorage/preview/2021-12-02/file.json

```diff
@@ -5871,16 +5887,13 @@
           }
         },
         "DirectoryPath": {
-          "type": "string",
-          "xml": {
-            "attribute": true
-          }
+          "$ref": "#/definitions/StringEncoded"
         },
         "Prefix": {
-          "type": "string"
+          "$ref": "#/definitions/StringEncoded"
         },
         "Marker": {
-          "type": "string"
+          "$ref": "#/definitions/StringEncoded"
```

Comment:
Probably want to revert this.  `StringEncoded` either applies to `NextMarker`, or doesn't apply to the markers in the response.